app/controllers/whats_app/webhook_controller.rb

```diff
@@ -149,7 +149,7 @@ def fetch_message_from_twilio(twilio_message_sid, organization)
 
     def handle_freeform_message_not_allowed_error(contributor, twilio_message_sid)
       message_text = fetch_message_from_twilio(twilio_message_sid, @organization)
-      message = Message.find_by(text: message_text)
+      message = @organization.messages.find_by(text: message_text)
```

Comment:
Looking through the code in this webhook, I think we could refactor the use of guard clauses, but it has nothing to do with the scope of this PR